src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,20 @@
+import styles from './Dropdown.module.css';
+
+export default function Dropdown({ text }) {
+
+  return (
+    <label className={styles.label}>
+      <div className={styles.container}>
+        <div className={styles.text}>{text}</div>
+        <select className={styles.select}>
+          <option className={styles.NotSubmitted}>Not Submitted</option>
+          <option className={styles.Submitted}>Submitted</option>
+          <option className={styles.NoCallback}>No Callback</option>
+          <option className={styles.InterviewScheduled}>Interview Scheduled</option>
+          <option className={styles.Interviewing}>Interviewing</option>
+          <option className={styles.OfferReceived}>Offer Received</option>
```

Comment:
The options here are hard-coded for what we'll use them for, but ideally the Dropdown component accepts three things:

1. The options to display
2. The selected option
3. A method to set the selected options

That way the component can be used for _any_ scenario.